irispy/new_sji.py

```diff
@@ -204,14 +205,82 @@ def apply_exposure_time_correction(self, undo=False, force=False):
                                                                    self.missing_axis))
 
 
-def read_iris_sji_level2_fits(filename, memmap=False):
+class IRISMapCubeSequence(NDCubeSequence):
+    """Class for holding, slicing and plotting IRIS SJI data.
+
+    This class contains all the functionality of its super class with
+    some additional functionalities.
+
+    Parameters
+    ----------
+    data_list: `list`
+        List of `IRISMapCube` objects from the same OBS ID.
+        Must also contain the 'detector type' in its meta attribute.
+
+    meta: `dict` or header object
+        Metadata associated with the sequence.
+
+    common_axis: `int`
+        The axis of the NDCubes corresponding to time.
+
+    """
+    def __init__(self, data_list, meta=None, common_axis=0):
+        # Check that all SJI data are coming from the same OBS ID.
+        if len(np.unique([cube.meta["OBSID"] for cube in data_list])) != 1:
```

Comment:
This could be rewritten as:
```python
if any([cube.meta["OBSID"] for cube in data_list] != data_list[0].meta["OBSID"]):
```
The list might have to be an array.  I'm not sure.  Check this out.

Reply:
With this line, I get an error that I solve by doing :
<if np.any([cube.meta["OBSID"] != data_list[0].meta["OBSID"] for cube in data_list]):>